Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -110,6 +112,24 @@ - (dispatch_queue_t)methodQueue
     };
   }
 
+  /*
+   * The `anchor` option takes a view to set as the anchor for the share
+   * popup to point to, on iPads running iOS 8. If it is not passed, it
+   * defaults to centering the share popup on screen without any arrows.
+   */
+  if ([share respondsToSelector:@selector(popoverPresentationController)]) {
+    share.popoverPresentationController.sourceView = ctrl.view;
+    NSNumber *anchorViewTag = [RCTConvert NSNumber:options[@"anchor"]];
+    if (anchorViewTag) {
+      UIView *anchorView = [self.bridge.uiManager viewForReactTag:anchorViewTag];
```

Comment:
I tried that - but here are the meanings of both of them:

* `sourceView`: The view containing the anchor rectangle for the popover.
* `sourceRect`: The rectangle in the specified view in which to anchor the popover.

So `sourceView` is actually some parent inside which the actual rectangle is there to which we have to point to. The arrow position comes out correctly only when I do it like this.